irods/test/scripts/test002.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+. "$BATS_TEST_DIRNAME"/funcs
+PYTHON=python3
+
+# Setup/prerequisites are same as for login_auth_test.
+# Run as ubuntu user with sudo; python_irodsclient must be installed (in either ~/.local or a virtualenv)
+#
+
+PASSWD=test123
+NEW_PAM_PASSWD="new_pass"
+
+setup()
+{
+    setup_pam_login_for_alice $PASSWD
+}
+
+teardown()
+{
+    finalize_pam_login_for_alice
+    test_specific_cleanup
+}
+
+@test create_secrets_file {
+
+    sudo chpasswd <<<"alice:$NEW_PAM_PASSWD"
+    rm -f ~/.irods/.irodsA
+    $PYTHON -c "import irods.client_init; irods.client_init.write_pam_credentials_to_secrets_file('$NEW_PAM_PASSWD')"
```

Comment:
What happens if `.irodsA` already exists?
Is that case already covered by another test?

Reply:
Not yet covered by a test.  Will decide later perhaps if it makes sense to overwrite an .irodsA.  Since iinit will do so when the pam credentials are expired, perhaps here it also makes sense to do so.  Thoughts?

Reply:
Yeah, let's discuss that at a later date and get this PR merged.

Open an issue for that, link it here, and mark this conversation as resolved.

Reply:
I see, looking back at the function `write_pam_credentials_to_secrets_file`, that I'm asserting the attribute naming file being null (synonymous with its not having been used) as a guarantee against rewriting an existing file, but that is in fact no guarantee.  Apart from everything else, someone could be running in an interpreter with -O (optimization active), so... I'd best remove that and then we'll be unconditionally overwriting a pre-existing .irodsA.  Then I'll make the issue as you asked and cross link here.

Reply:
Delaying conversation and fix until v3.0.0 . See #635